src/pages/api/sendEmail.ts

```diff
@@ -0,0 +1,48 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import nodemailer from 'nodemailer';
+
+type Data = {
+  message?: string;
+  error?: string;
+  details?: string;
+};
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  const { to, subject, text } = req.body;
+
+  // Validate the input
```

Comment:
Do you it makes sense to use a validation library like [Zod](https://zod.dev/) or [Joi](https://joi.dev/api/?v=17.13.3) instead of writing this ourselves?

Reply:
From my understand, these emails will sent based off the email in the User object. So the validation should happen there so that there are no invalid emails in our database. So I don't think any validation needs to happen here. Lmk if you think otherwise.

Reply:
Apologies for replying late.
- imo ideally some basic validation would be a good idea as a sanity check. If we ever write incorrect code such that the DB isn't populated correctly, being able to detect it here via a log message sounds useful to me. But this just an opinion, not a fact.

- Could you please tell me whether the following is correct?
Eventually, we'll be sending emails from a server side piece of code only. So, the current handler function won't be called by the client-side code directly.

Reply:
Talked to Chinemerem. My bad. Don't need to deal with this rn. Will deal with this issue when we're integrating this code. Thank you Kelly!